docs/operating-systems/course-tools.md

```diff
@@ -148,13 +148,13 @@ If you haven’t installed gatorgrade type this command:
 
 ### Go installation process:
 
-If you haven't installed go click the link and follow [the instructions](https://go.dev/dl/)
+If you haven't installed Go click the link and follow [the instructions](https://go.dev/dl/)
 
 Make sure you click the right chip that your laptop has.
 
 If you choose the right version and it still isn't working, follow these instructions:
-1. type in the command ```nano ~/.zshrc``` into your terminal
-2. add in the following 2 paths into the file that pops up in your terminal:
+1. Type in the command ```nano ~/.zshrc``` into your terminal
+2. Add in the following 2 paths into the file that pops up in your terminal:
   - ```export PATH=$PATH:/usr/local/go/bin```
   - ```export PATH=$PATH:$GOPATH/bin```
 3. After you save the changes in that file, source the file using the command ```. ~/.zshrc```.
```

Comment:
This whole section has wonky formatting, it is in a paragraph rather than a list. Can you change it to look the correct way?

Reply:
Also, you don't use a numbered list for any of the rest of this page so far. Maybe you should change it to match the rest of the file so far?